frontend/src/components/ModelBox/ModelSelection.tsx

```diff
@@ -80,8 +87,8 @@ function ModelSelection({ chatModelOptions }: { chatModelOptions: string[] }) {
 
 				<div className="model-selection-info">
 					{errorChangingModel ? (
-						<p className="error">
-							Could not change model. You are still chatting to:
+						<p className="error-message" aria-live="polite">
```

Comment:
While I understand you want to use the same styles in two different places (here and in DefenceConfigurationInput), it's usually better to have css defined alongside its component. This class is currently defined in ControlPanel.css.

I was going to suggest extracting a component for this error message, and defining the style there instead, however, that feels a bit overkill as I assume you only expect one style of error message within the whole Control Panel component hierarchy.

The other possibility would be to duplicate the css class and declare the styles in both places. To be honest, as it's only two places you're using it currently, and it's only two styles (one of which uses a theme property), I'm inclined to suggest that instead of tucking it away in ControlPanel.css. But then make the selectors more specific than just `.error-message`, as suggested elsewhere.